neo/Consensus/ConsensusService.cs

```diff
@@ -424,9 +445,12 @@ private void OnPrepareRequestReceived(ConsensusPayload payload, PrepareRequest m
                         context.PreparationPayloads[i] = null;
             context.PreparationPayloads[payload.ValidatorIndex] = payload;
             byte[] hashData = context.MakeHeader().GetHashData();
+            byte[] stateData = context.MakeStateRoot().GetHashData();
```

Comment:
We need to verify the StateRootSignature of the Primary, no?

Reply:
Verified here:
https://github.com/neo-project/neo/blob/445c9900ad62ad2caf9ac5ad8ccbc7cc4e73d9d7/neo/Consensus/ConsensusService.cs#L424-L431

But didn't re-use `stateRootHashData`. Will fix.

Reply:
#1809

Reply:
Merged